designer/server/src/plugins/errorPage.ts

```diff
@@ -20,12 +26,14 @@ export default {
           // An error was raised during
           // processing the request
           const statusCode = response.output.statusCode
-          const pageTitle = 'You do not have access to this service'
 
-          // In the event of 403
-          // return the `403` view
-          if (statusCode === 403) {
-            return h.view('403', errorViewModel(pageTitle)).code(statusCode)
+          if (statusCode in errorCodes) {
```

Comment:
Using a map everything works

```suggestion
          const errorMessage = errorCodes.get(statusCode)

          if (errorMessage) {
```